spec/models/miq_policy_spec.rb

```diff
@@ -100,6 +100,16 @@
       end
     end
 
+    describe '#miq_event_definitions' do
+      before do
+        policy.miq_policy_contents.push(FactoryGirl.create(:miq_policy_content))
+      end
+
+      it 'lists event definition' do
+        expect(policy.miq_event_definitions).to eq([event])
```

Comment:
I think `eq` is more stricter, shorter, and simpler. 

My reading of `match_array` is that it is useful only when we are unsure about array ordering.